jaeger-core/src/main/java/io/jaegertracing/internal/JaegerSpan.java

```diff
@@ -74,6 +75,10 @@ public long getStart() {
     return startTimeMicroseconds;
   }
 
+  public boolean isFinished() {
+    return finished;
+  }
```

Comment:
why is this needed?

Reply:
We use a custom async/future framework at my work. We pass spans to the framework which will activate the span on ThreadLocal scope when futures will be executed. 
Sometimes the futures will have stale or old spans which are not cleaned up properly (mostly due to misuse of the framework, or cached futures...etc). 
I have a custom ScopeManager which detects a span isFinished() but yet will be activated on a ThreadLocal scope. I use a counter metric to record when this occurs. 
Without this method available, I would (have) never know this even occurred.

Reply:
Could you please create a separate issue to discuss this, and not include it in this PR?

Reply:
sure thing